JavaFiddle-ejb/src/main/java/ru/javafiddle/core/ejb/LibraryBean.java

```diff
@@ -43,42 +43,43 @@ public void add(String projectHash, String libraryName) {
     }
 
     public List<String> getAll(String projectHash) {
-
-        Hash hash = (Hash)em.createQuery("SELECT h FROM Hash h where h.hash =:projecthash")
-                .setParameter("projecthash", projectHash)
-                .getSingleResult();
-
-        Project project = hash.getProject();
-
-        List<LibraryToProject> libs = project.getLibraries();
-        List<String> libNames = new LinkedList<>();
-
-        for (LibraryToProject l:libs) {
-
-            libNames.add(l.getLib().getLibraryName());
-        }
-
-        return libNames;
-
-
+//!TODO
+//        Hash hash = (Hash)em.createQuery("SELECT h FROM Hash h where h.hash =:projecthash")
+//                .setParameter("projecthash", projectHash)
+//                .getSingleResult();
+//
+//        Project project = hash.getProject();
+//
+//        List<LibraryToProject> libs = project.getLibraries();
+//        List<String> libNames = new LinkedList<>();
+//
+//        for (LibraryToProject l:libs) {
+//
+//            libNames.add(l.getLib().getLibraryName());
+//        }
+//
+//        return libNames;
+
+    return null;
```

Comment:
It should be marked as "TODO". Couse of some changes in mapping this method needs to be refactored. I simly have no time yet to accomplish this. Shall I add this mark explicitly?